src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -123,9 +124,9 @@ def get(self, request, organization, trace_id):
             if len(result["data"]) == 0:
                 return Response(status=404)
             len_transactions = len(result["data"])
-            sentry_sdk.set_tag("discover.trace-view.num_transactions", len_transactions)
+            sentry_sdk.set_tag("trace-view.num_transactions", len_transactions)
             sentry_sdk.set_tag(
-                "discover.trace-view.num_transactions.grouped",
+                "trace-view.num_transactions.grouped",
```

Comment:
Will we settle on a convention of using just `-` or `_` rather than mixing the two together?

Reply:
lol `we`, I think I'm the only one who keeps mixing them, I'll change this to `_` since that's what most of our tags use